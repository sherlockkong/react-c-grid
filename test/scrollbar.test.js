import React from 'react';
import Scrollbar from '../src/components/Scrollbar';
import renderer from 'react-test-renderer';

it('scrollbar', () => {
	const scrollbar = renderer
		.create(<Scrollbar onScroll={(left) => { }}>Grid Content</Scrollbar>)
		.toJSON();
	expect(scrollbar).toMatchSnapshot();
});