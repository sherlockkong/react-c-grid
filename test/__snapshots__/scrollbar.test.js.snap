// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`scrollbar 1`] = `
<div
  style={
    Object {
      "height": "100%",
      "overflow": "hidden",
      "position": "relative",
      "width": "100%",
    }
  }
>
  <div
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onScroll={[Function]}
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "marginBottom": 0,
        "marginRight": 0,
        "overflow": "scroll",
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    Grid Content
  </div>
  <div
    className="vertical-scrollbar-container"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseMove={[Function]}
  >
    <div
      className="vertical-scrollbar scrollbar"
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseMove={[Function]}
    />
  </div>
  <div
    className="horizontal-scrollbar-container"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseMove={[Function]}
  >
    <div
      className="horizontal-scrollbar scrollbar"
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseMove={[Function]}
    />
  </div>
</div>
`;
